net.sourceforge.electrip.Electrip.json

```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "net.sourceforge.electrip.Electrip",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "40",
+    "runtime-version": "42",
```

Comment:
43 is out.

Reply:
Ah, thanks, thought I changed that already.